docs/xks/developer-guide/scheduling-scaling.md

```diff
@@ -0,0 +1,254 @@
+
+## Pod Resources
+
+Setting the correct [resources for a Pod](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) is an important component in helping the scheduler select the correct Node to schedule a Pod to. There are two types of resource configuration fields, and these are configured for each container in a Pod.
+The resource request tells the scheduler how much of each resource each container in a Pod is expected to consume, it is however allowed of a Pod to consume more. The limit sets the maximum amount of resources each container in a Pod can consume. There are two main resource types that one should be aware of, CPU and memory.
+The CPU resource is defined in CPU units where one CPU unit is the equivalent of one CPU core. Fractional units can also be requested, in either decimals like `0.1` but also in terms of millicores where `100m` would be the same amount. The memory resource is defined in whole integers with the quantity suffixes
+`Ei, Pi, Ti, Gi, Mi, Ki`.
+
+If you do not specify any resources for a container the default resource request and limit will be applied as shown below. These resources are low on purpose, both to minimize the effects of over provisioning but also to make it obvious for end users that resources have not been specified. The keen-eyed may notice that
+a CPU limit is not set by default, this is on purpose and the reasons for it will be discussed later.
+
+```yaml
+resources:
+  request:
+    cpu: 50m
+    memory: 32Mi
+  limits:
+    memory: 256Mi
+```
+
+The scheduler will look at the cumulative resource requests across all containers in a Pod during the scheduling phase. The scheduler will exclude any Node which does not have capacity for the Pods resource request. Capacity is determined based on the total resources available in a Node minus the sum of all the
+resource requests of all Pods currently scheduled to the Node. This makes it possible to over provision as the resource request can be much larger than the actual consumption of the Pod. Efficient resource allocation is a constant battle between requesting enough resources to avoid under allocation while not
+requesting too much which would result in over allocation. A Pod may at times request more resource than any Node has capacity for, there are two possible outcomes for this situation. If the Pods resource request is less than a Nodes total available resources, a new Node will be added to the cluster. The Pod will

```

Comment:
"...than any existing Node can accommodate, ..."
"If the Pod's resource request ..."
Can I suggest instead "A new node will be started to accommodate the Pod, unless the resource request exceeds the total available resources. In this case, the pod will be considered unschedulable." ?